src/plugins/initiatives/initiativesDirectory.test.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import {fromJSON, toJSON} from "./initiativesDirectory";
+
+const exampleInitiativeFile = () => ({
```

Comment:
nit: personally, I would make this a const and `deepFreeze` it. Change not requested (since it's not particularly better either way).

Also, I would add the explicit type signature, just as an added safety in case we add/remove fields. (Although there'd still be a type error at the toJSON call, that error would be less 'direct').

Reply:
Added the return type. But will keep the generator function.
In a follow-up we'll have a generator besides this which takes arguments :]
Also find that generator functions make this very pleasant to read.
```js
// Given
const initiativeFile = exampleInitiativeFile();
```